src/DDTrace/Integrations/CodeIgniter/CodeIgniterIntegration.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace DDTrace\Integrations\CodeIgniter;
+
+use DDTrace\Integrations\Integration;
+
+class CodeIgniterIntegration extends Integration
+{
+    const NAME = 'codeigniter';
+
+    /**
+     * @return string The integration name.
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * Static method to add instrumentation to PDO requests
+     */
+    public static function load()
+    {
+        /* This is just a dummy integration to appease tests that expect all
```

Comment:
I think this is fine for now (and for WP integration), but let's fix this before the next release. :)